app.py

```diff
@@ -56,6 +63,26 @@ def get_by_id(id_):
     except Exception as e:
 	    return(str(e))
 
+@app.route("/members_by_state/<state_>")
+def get_members_by_state(state_):
+    headers = {'X-API-Key': os.getenv('PROP_API')}
+    URL = f'https://api.propublica.org/congress/v1/members/senate/{state_}/current.json'
+    response = requests.get(URL, headers = headers).json()
+    results = response['results']
+
+    objects = map(lambda result: MemberIndex(first_name = result['first_name'],
+                                             party = result['party'],
```

Comment:
Possibly add house reps to the same method to get them all in one endpoint. We should look into services but the way modules are called still confuse me. I think all you need is "import"? I wonder how it works when its nested in folders. Something to research...

Reply:
I agree, it'd be nice to get them all in one call.